Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM php:7.1-apache
```

Comment:
So, what's the requirement about PHP version here?
If a user has installed PHP via normal procedures of his/her environments, we should just run "php examples/BasicOperations/GetCampaigns.php"?

Reply:
The PHP version is actually specified inside the container, so we can fix it to the one we like; in this case it's 7.1 since it's the minimum required, but we can also use a more recent version.

In case a user has already installed PHP on their machines, the docker environment will not override it, but they will be able to run `php examples/BasicOperations/GetCampaigns.php` inside the container with the PHP version specified by the container itself.

They can also run it outside of the container in their local machines of course, provided they have all the dependencies installed.

Reply:
I hit a problem first because my machine doesn't have PHP installed in a traditional way, so there was an error saying that it couldn't find php71 binary file.
Could you add a few sentences in README.md explaining about this?
It's probably should be a knowledge of using Docker, but it's more smooth to warn the users in advance.

Reply:
When did you get the error? When running `docker-compose up` or when trying to execute the examples? Not really sure what was happening when you got the error, I may need some more info to clarify this in the README.

Reply:
My error was because I set some alias for the php binary itself.
So it'll be an error like "bash: xxxxx/php: No such file or directory"
I think it's a rare case, so probably we don't need to care much now.

Probably, if you know where Docker will install PHP (perhaps, /usr/local/bin/php), you may just say so briefly.

Reply:
This is kind of weird as I already had PHP installed on my machine when I tried it and also, AFAIK, the Docker image will take care of installing the interpreter in the container on its own, regardless of the existence or not of the interpreter in the host machine.

Reply:
Yes, but I guess you didn't add strange alias that would hide the originally installed PHP right?

If you're sure that Docker will install its own PHP binary and will work out of the box unless someone sets the alias that would hide it, then it's fine now.
Could you finally confirm this?

Reply:
Yes, I'm positive about it. Let's merge this, then.